share/picongpu/dockerfiles/ubuntu-1604/modules.yaml

```diff
@@ -5,7 +5,14 @@ modules:
     openmpi:
       environment:
         set:
-          OMPI_MCA_mpi_leave_pinned: '0'
+          OMPI_MCA_btl: '^vader'
+    # This anonymous spec selects any package that
+    # depends on openmpi. The double colon at the
+    # end clears the set of rules that matched so far.
+    ^openmpi::
+      environment:
+        set:
+          OMPI_MCA_btl: '^vader'
```

Comment:
We should be able to use the latest MPI with `OMPI_MCA_btl_vader_single_copy_mechanism=none` to disable vader: https://github.com/open-mpi/ompi/issues/4948#issuecomment-377341406